lua/typescript-tools/tsserver_provider.lua

```diff
@@ -106,21 +106,19 @@ end
 
 ---@return Path|nil
 local function get_tsserver_from_mason()
-  local ok, mason_registry = pcall(require, "mason-registry")
+  local MASON = os.getenv "MASON"
```

Comment:
I swear one last thing! 😄
```suggestion
  local mason_path = os.getenv "MASON"
```

Reply:
done 😄